src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -38,4 +46,20 @@ public static void printHelp() {
                 "Go back to the workout menu."));
         printer.showToUser(helpMessage.toString().trim());
     }
+
+    public static void emptyInputError() {
+        System.out.println("Please enter something.");
+    }
+
+    public static void inputPrompt() {
+        System.out.print("Workout Session >>>>> ");
```

Comment:
Workout Session is under Workout Menu, so is it clearer if we show "Workout Menu > Workout Session >>>>>" ?

Reply:
great idea